code/modules/mob/living/simple_animal/abnormality/teth/spider_bud.dm

```diff
@@ -38,21 +38,68 @@
 	observation_fail_message = "I devoured the prey. <br>\
 		My body reacted faster than my thoughts. <br>... <br>I am a spider. <br>I eat anything my web catches."
 
+	/// Filled with ckeys of people who broke our cocoons, they need to pay if they dare mess with us
+	var/list/metagame_list = list()
```

Comment:
You can make this static so its consistantly shared between all spider buds. IF thats the intention. Since spiderbud cant die unless a admin kills them i assume this is essentially a MANUAL RESET.

Reply:
I think its fine, the only way spiderbud dies would be via player sentience or admin boose